meta-refkit-extra/doc/bluetoothspeaker.rst

```diff
@@ -0,0 +1,73 @@
+Bluetooth Speaker
+#################
+
+This is a demo to convert Intel 570x development board into stand
+alone bluetooth speaker. Main application is a python file
+``btspeaker.py`` and it is run under a Linux user called
+``btspeaker``. The demo uses fairly "stock" components like
+``PulseAudio``, ``BlueZ`` and ``ConnMan``. These are included in the
+IoT Reference OS Kit's gateway profile. The 570x doesn't have audio
+codec or speaker included so if you want to hear actual sound you need
+to connect something either to HDMI (like home theater amplifier) or
+to USB (like USB sound card or headset).
+
+``btspeaker.py`` is started in boot with ``systemd`` service file and
+``PulseAudio`` is started at the same time by playing a "startup
+tune". There's also special ``PulseAudio`` ``client.conf`` installed to
+btspeaker's home directory to make ``PulseAudio`` not exit when idling
+(this is the default ``PulseAudio`` behavior). This is done because we
+want to listen and wake up for ``BlueZ`` audio connection events. With
+these events ``PulseAudio`` setups the A2DP audio path. Also in the
+beginning of the execution ConnMan is used through ``DBus`` to unblock
+the bluetooth radio.
+
+How to build
+============
+
+1) Uncomment the following line in refkit-extra.conf:
```

Comment:
For the sake of completeness, yes. If the user strictly follows this doc nothing gets built.